services/src/atdd-staging/src/main/resources/features/flow_crud.feature

```diff
@@ -15,23 +15,24 @@ Feature: Flow CRUD
     And each flow has unique flow_id
     And each flow has max bandwidth set to 10000
 
-    When creation request for each flow is successful
+    When initialize creation of given flows
 
     Then each flow is created and stored in TopologyEngine
     And each flow is in UP state
     And each flow can be read from Northbound
-    And each flow has rules installed
+    And each flow is valid per Northbound validation
+    And all active switches have correct rules installed per Northbound validation
     And each flow has meters installed with 10000 max bandwidth
     And each flow has traffic going with bandwidth not less than 10000
 
     Then each flow can be updated with 5000 max bandwidth
     And each flow is in UP state
-    And each flow has rules installed
```

Comment:
Shouldn't we check "all active switches have correct rules installed per Northbound validation" here?